src/GraphNodeManagerViewExtension/GraphNodeManagerViewExtension.csproj

```diff
@@ -1,79 +1,38 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
hmm...I must have missed this project when first moving to sdk style ?

Reply:
I think maybe it was not merged yet.